py/CHANGELOG.md

```diff
@@ -3,9 +3,12 @@
 ## Unreleased
 
 - Add `allow_negative` to `BuiltinMeasurementKey`. Filter out negative BuiltinMeasurements if `allow_negative` is false. ([#2982](https://github.com/getsentry/relay/pull/2982))
+- Add ability to block metric tags matching a glob pattern. ([#2973](https://github.com/getsentry/relay/pull/2973))
 
 ## 0.8.44
 
+- Add ability to block metrics matching a glob pattern. ([#2954](https://github.com/getsentry/relay/pull/2954))
```

Comment:
so this was added together with the 'various fixes and improvements', but this is more of a new feature. Not sure how to best retroactively add it

Reply:
You did it right IMO. "Various fixes & improvements" seems to be a bot thing.